.github/workflows/cache-issues.yaml

```diff
@@ -0,0 +1,29 @@
+name: Report remaining open issues
+on:
+  schedule:
+    # Daily at 8:20 UTC
+    - cron: '20 8 * * *'
+  pull_request:
+    branches:
+     - master
+  push:
+    branches:
+      - master
+jobs:
+  track_pr:
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          mkdir -p issues
+          gh issue list --state all --json number,state,labels --limit 30000 --template '{{range .}}{{.number}},{{.state}},{{range .labels}}{{.name}} {{end}} {{end}}'' --repo scylladb/scylladb > issues/scylladb_scylladb.csv
+          gh issue list --state all --json number,state,labels --limit 30000 --template '{{range .}}{{.number}},{{.state}},{{range .labels}}{{.name}} {{end}} {{end}}' --repo scylladb/scylla-enterprise > issues/scylladb_scylla-enterprise.csv
+          gh issue list --state all --json number,state,labels --limit 30000 --template '{{range .}}{{.number}},{{.state}},{{range .labels}}{{.name}} {{end}} {{end}}' --repo scylladb/scylla-cluster-tests > issues/scylladb_scylla-cluster-tests.csv
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Upload file to bucket
+        uses: koraykoska/s3-upload-github-action@master@0.1.3
+        env:
+          FILE: ./issues/
```

Comment:
will it be publicitly available?

Reply:
no, still need to figure credentials, since it's not running in Jenkins.